bin/ons_csv_to_ctb_json_main.py

```diff
@@ -127,6 +127,14 @@ def main():
                              '"--dataset-filter TS,RM" to include datasets with a '
                              'Dataset_Mnemonic beginning with either TS or RM.')
 
+    parser.add_argument('--base-dataset-name',
+                        type=str,
+                        default='base',
+                        help='Name to use for base dataset which contains all variabless and '
+                             'which all other datasets include. The purpose of this dataset is '
+                             'to minimise the size of the dataset JSON file by avoiding '
```

Comment:
Is it worth mentioning that the base dataset does not have to really exist as a Cantabular dataset?